zulip_bots/zulip_bots/bots/monkeytest/monkeytest.conf

```diff
@@ -0,0 +1,2 @@
+[monkeytest]
+key=12345
```

Comment:
Make sure you include a newline at the end of this file so that little red sign goes away! :)